sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -386,7 +386,19 @@ def server_name_from_env
       end
     end
 
-    # allow extensions to extend the Configuration class
-    def post_initialization_callback; end
+    def post_initialization_callback
```

Comment:
since there are now multiple callbacks to be run, this should be renamed to `run_post_initialization_callbacks`

Reply:
Should I define an alias for this method too, so older versions of `sentry-rails` could work with newer version of `sentry-ruby`?

Reply:
no it won't be necessary. we've restricted the dependency to `~> 4.1.2` so users will need to upgrade `sentry-rails` for `4.2` as well.